### PR TITLE
common: a small modifycation about getting rid of unneccessary contio…

### PR DESCRIPTION
**Pull request by shun-s**
…n judgement in loop while

a small modifycation about getting rid of unneccessary contion judgement in loop while

Signed-off-by: shun-s <song.shun3@zte.com.cn>

### CODE REVIEW

File: src/common/config.cc [hunk 1]
```diff
@@ -689,7 +689,7 @@ int md_config_t::injectargs(const std::string& s, std::ostream *oss)
   char *p = b;
   while (*p) {
     nargs.push_back(p);
-    while (*p && *p != ' ') p++;
+    while (*p != ' ') p++;
```
Comment (tchaikov) [inline src/common/config.cc]: `(*p != '\0' && *p != ' ')` and `(*p != ' ')` is not equivalent.

Comment (shun-s) [inline src/common/config.cc]: yeah, it's not equivalent, but it seems to have nothing to do with that.
As (*p != '\0' && *p != ' ') is always running after while(*p) , so " *p != '\0' " is always true. there is no necessary to judge whether *p != '\0' is true again.

Comment (tchaikov) [inline src/common/config.cc]: please note `p++` after `while (*p && *p != ' ')`

**Comment by tchaikov** [comment]
i am closing this PR as it does not sound right.

File: src/common/config.cc (hunk 1 above)
Comment (shun-s) [inline src/common/config.cc]: yeah, you are right, thank you  ^^

**Pull request closed without merging**